packages/bun-types/events.d.ts

```diff
@@ -121,7 +121,11 @@ declare module "events" {
      * @param eventName The name of the event.
      * @param listener The callback function
      */
+<<<<<<< HEAD
```

Comment:
Merge conflict

Reply:
how do i resolve it? since i installed all the modules used by Bun this has been occuring